waiter/src/waiter/token.clj

```diff
@@ -23,20 +22,25 @@
 (def ^:const ANY-USER "*")
 (def ^:const valid-token-re #"[a-zA-Z]([a-zA-Z0-9\-_$\.])+")
 
-(let [default-etag 0]
-  (defn- token-metadata->etag
-    "Converts the token metadata to a tag"
-    [token-metadata]
-    (-> token-metadata
-        (get "last-update-time")
-        (or default-etag)
-        str)))
+(let [default-etag "0"]
```

Comment:
Why does our default ETag not start with an `"E"`?  Why are we putting `"E"` in front of ETags generally?

Reply:
I placed E, so that it is easy to identify them vs just seeing a random hash. I will add the `E` prefix to the default etag.